File: play-2.5.x/K-Modules/kmodules/K-Base/app/K/Aop/annotations/DBIndexed.java
package K.Aop.annotations;

import java.lang.annotation.Retention;
import java.lang.annotation.Target;

import static java.lang.annotation.ElementType.FIELD;
import static java.lang.annotation.ElementType.METHOD;
import static java.lang.annotation.RetentionPolicy.RUNTIME;

/**
 * Created by kk on 14/11/14.
 */

@Target({METHOD, FIELD})
@Retention(RUNTIME)
public @interface DBIndexed {

}

File: play-kotlin-2.5.x/PlayKotlin/kmodules/K-Base/app/K/Aop/annotations/DBIndexed.kt
package K.Aop.annotations

import java.lang.annotation.Retention

import java.lang.annotation.ElementType.FIELD
import java.lang.annotation.ElementType.METHOD
import java.lang.annotation.RetentionPolicy.RUNTIME

/**
 * Created by kk on 14/11/14.
 */

@Target(AnnotationTarget.FUNCTION, AnnotationTarget.PROPERTY_GETTER, AnnotationTarget.PROPERTY_SETTER, AnnotationTarget.FIELD)
@Retention(RUNTIME)
annotation class DBIndexed
